calico-enterprise/reference/component-resources/node/felix/configuration.mdx

```diff
@@ -109,6 +109,18 @@ The full list of parameters which can be set is as follows.
 
+### Go runtime configuration
+
+These configuration pararmeters allow configuration of the Go language runtime.
+They allow the `GOGC`, `GOMEMLIMIT` and `GOMAXPROCS` tunables to be accessed
+via Felix's configuration.
+
+| Configuration file parameter        | Environment variable                      | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        | Schema  |
+| ----------------------------------- | ----------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ------- | 
+| GoGCThreshold                       | `FELIX_GOGCTHRESHOLD`, `GOGC`             | Sets the Go runtime's garbage collection threshold; this is the percentage that the heap is allowed to grow before GC is triggered.   In general, doubling the value halves the CPU timespent doing GC, but it also doubles peak GC memory overhead.  A special value of -1 can be used to disable GC entirely; this should only be used in conjunction with the `GoMemoryLimitMB` setting.  This setting is overridden by the `GOGC` environment variable.                                                                                                                                                        | int     |
+| GoMemoryLimitMB                     | `FELIX_GOMEMORYLIMITMB`, `GOMEMLIMIT`     | GoMemoryLimitMB sets a (soft) memory limit for the Go runtime in MB.  The Go runtime will try to keep its memory usage under the limit by triggering GC as needed.  To avoid thrashing, it will exceed the limit if GC starts to take more than 50% of the process's CPU time.  A value of -1 disables the memory limit. Note that the memory limit, if used, must be considerably less than any hard resource limit set at the container or pod level.  This is because felix is not the only process that must run in the container or pod. This setting is overridden by the `GOMEMLIMIT` environment variable. | int     |
```

Comment:
```suggestion
| GoMemoryLimitMB                     | `FELIX_GOMEMORYLIMITMB`, `GOMEMLIMIT`     | GoMemoryLimitMB sets a (soft) memory limit for the Go runtime in MB.  The Go runtime will try to keep its memory usage under the limit by triggering GC as needed.  To avoid thrashing, it will exceed the limit if GC starts to take more than 50% of the process's CPU time.  A value of -1 disables the memory limit. Note that the memory limit, if used, must be considerably less than any hard resource limit set at the container or pod level.  This is because Felix is not the only process that must run in the container or pod. This setting is overridden by the `GOMEMLIMIT` environment variable. | int     |
```
Please make this change throughout.